packages/scalar-app/src/renderer/src/main.ts

```diff
@@ -100,4 +106,5 @@ async function drop(e: DragEvent) {
 function dragover(e: DragEvent) {
   e.preventDefault()
   e.stopPropagation()
```

Comment:
Optional but I _think_ you can remove `stopPropagation` since you added `stopImmediatePropagation`

Reply:
oh that's a testing leftover, removed!